### PR TITLE
fdroidserver 2.3.2 updates: puremagic overridden by libmagic

### PR DESCRIPTION
**Pull request by eighthave**
* libmagic overrides puremagic
* stop BrewBot from removing biplist and pycountry

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]
Thanks @eighthave!

File: pypi_formula_mappings.json [hunk 1]
```diff
@@ -317,8 +317,8 @@
     "exclude_packages": ["certifi"]
   },
   "fdroidserver": {
-    "exclude_packages": ["certifi", "cryptography", "numpy", "pillow"],
-    "extra_packages": ["androguard==3.3.5", "setuptools"]
+    "exclude_packages": ["certifi", "cryptography", "numpy", "pillow", "puremagic"],
+    "extra_packages": ["androguard==3.3.5", "biplist", "pycountry", "python-magic", "setuptools"]
```
Comment (branchvincent) [inline pypi_formula_mappings.json:321]: I'm assuming these extra packages enable optional features? Can fdroidserver define an extra instead so Homebrew (and any `pip` user) doesn't need to manually manage these?

Comment (eighthave) [reply pypi_formula_mappings.json:321]: Its already in _fdroidserver_:

```python
    extras_require={
        'optional': ['biplist', 'pycountry'],
```

I just don't know how to enable that in the Formula

Comment (branchvincent) [reply pypi_formula_mappings.json:321]: Awesome, so we can do:

```suggestion
    "package_name": "fdroidserver[optional]",   
    "exclude_packages": ["certifi", "cryptography", "numpy", "pillow", "puremagic"],
    "extra_packages": ["androguard==3.3.5", "python-magic", "setuptools"]
```

What do you think about also adding a `libmagic` extra (or similar) that depends on `python-magic`? That way it's more explicit that it's something fdroidserver supports and benefits more than just Homebrew (we try to keep `extra_packages` limited to temporary bugfixes)

Comment (eighthave) [reply pypi_formula_mappings.json]: Thanks, I included your proposal.  Including `python-magic` in `optional` makes sense and is easy.  Dealing with `libmagic` binaries there seems like a pain. I'd prefer to deal with it in the packaging for now at least.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**